src/util.cpp

```diff
@@ -737,31 +737,31 @@ fs::path GetConfigFile(const std::string& confPath)
 
 void ArgsManager::ReadConfigStream(std::istream& stream)
 {
-    if (!stream.good())
```

Comment:
Replace this with an assertion instead, even if just to self-document assumptions?